messages/index.js

```diff
@@ -549,6 +569,7 @@ module.exports = {
     cs: 'musí vyhovět regulárnímu výrazu "{{=e.params.pattern}}"',
     de: 'muss diesem Muster entsprechen: "{{=e.params.pattern}}"',
     es: 'debe coincidir con el patron "{{=e.params.pattern}}"',
+    fi: 'täytyy vastata mallia "{{=e.params.pattern}}"',
```

Comment:
is there a better word for regular expression or is it what should be used?

Reply:
I think "muotoa" would be better. It's basically same as "format" or "pattern" in this occasion.

Reply:
Changed for "muotoa".